### PR TITLE
Fix toggle for including sample content in template

### PR DESCRIPTION
**Pull request by jfversluis**
### Description of Change

This fixes the toggle for including sample content in the template. In #25419 some last-minute changes were made to not make the sample content show up by default, but that introduced a bug where you couldn't get the sample content to show up at all using Visual Studio.

With this change the sample content can optionally be included again in Visual Studio by using the checkbox and through the CLI by specifying either `--sample-content` or `-sc`.

Also the not used `Empty` switch was removed as well as the `SampleContent` variable which did not serve any additional purpose.

### CODE REVIEW

File: src/Templates/src/templates/maui-mobile/.template.config/dotnetcli.host.json [hunk 1]
```diff
@@ -4,11 +4,9 @@
       "Framework": {
         "longName": "framework"
       },
-      "Empty": {
-        "longName": "empty"
-      },
       "IncludeSampleContent": {
-        "isHidden": true
+        "longName": "samplecontent",
```
Comment (Redth) [inline src/Templates/src/templates/maui-mobile/.template.config/dotnetcli.host.json:8]:
```suggestion
        "longName": "sample-content",
```

**Review by Redth — DISMISSED** [review]

**Comment by mrlacey** [comment]
Will this be available in VS before the GA release?

**Pull request merged**